priv/site/schedule.slime

```diff
@@ -0,0 +1,134 @@
+---
+layout: _layout.slime
+---
+
+.Schedule
+  = include(@env, "_includes/schedule/nav.slime")
+  .Schedule-hero
+    .Schedule-heroImage
+      img src="public/schedule_hero_image.jpg"
+
+    .Schedule-content
+      .Schedule-contentDateTime
+        time datetime="2021-05-28"
+          | May 28 2021
+        p 09:00 - 18:30
+
+      h1.Schedule-contentTitle Schedule
+
+      ul.Schedule-list
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+        li
+          span.Schedule-listTime 9:00-9:15
+          span.Schedule-listTalkTitle Introduction
+          span.Schedule-listSpeaker Alchemy Team
+
+  .Marquee
+      = for _ <- 1..2 do
+        div
+          = for _ <- 1..2 do
+            span Talks
+            span 2021
+
+section.Speakers
```

Comment:
Shouldn't this be name after `Schedule`? Are you reusing any styles from `Speakers` here?